src/client/vib.ts

```diff
@@ -63,15 +63,15 @@ class VIB {
     try {
       core.debug(`Sending pipeline [pipeline=${util.inspect(pipeline)}]`)
 
-      const response = await this.pipelinesClient.startPipeline(pipeline, {
+      const response: AxiosResponse = await this.pipelinesClient.startPipeline(pipeline, undefined, {
```

Comment:
There is a long time I do not use node, but checking if `undefined == null` is really weird for me. I would suggest changing that `undefined` to `null`